src/main/java/com/projetoGerenciamentoCurso/gerenciamentoCurso/service/AlunoService.java

```diff
@@ -1,36 +1,83 @@
 package com.projetoGerenciamentoCurso.gerenciamentoCurso.service;
 
+import javax.transaction.Transactional;
+
 import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import com.projetoGerenciamentoCurso.gerenciamentoCurso.*;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.GerenciamentoCursoApplication;
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.dto.AlunoDto;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.models.Aluno;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.repository.AlunoRepository;
 
 import lombok.AllArgsConstructor;
 
 @Service
 @AllArgsConstructor
+@Transactional
 public class AlunoService {
 
 	private final RabbitTemplate rabbitTemplate;
-
-	public void retornaAluno(AlunoDto aluno) {
+	
+	@Autowired
+	private AlunoRepository alunoRepository;
+	
+	public Aluno retornaAluno(AlunoDto aluno) {
 
 		rabbitTemplate.convertAndSend(GerenciamentoCursoApplication.EXCHANGE_NAME,
 				GerenciamentoCursoApplication.ROUTING_KEY, aluno);
+		
+		Aluno alunoModel = alunoRepository.save(criarAlunoModel(aluno));
+		
+		return alunoModel;
 
 	}
 
-	public void deletarAluno(AlunoDto aluno) {
+	public Aluno deletarAluno(AlunoDto aluno) {
 
 		rabbitTemplate.convertAndSend(GerenciamentoCursoApplication.EXCHANGE_NAME,
 				GerenciamentoCursoApplication.ROUTING_KEY_ALUNO_DELETAR, aluno);
+		
+		Aluno alunoModel = alunoRepository.getOne(aluno.getMatricula());
+		if(alunoModel!=null) {
+			 
+			alunoRepository.delete(alunoModel);
+		}
+		
+		return alunoModel;
 	}
-
-	public void atualizarAluno(AlunoDto aluno) {
+	
+	public Aluno atualizarAluno(AlunoDto aluno) {
 
 		rabbitTemplate.convertAndSend(GerenciamentoCursoApplication.EXCHANGE_NAME,
 				GerenciamentoCursoApplication.ROUTING_KEY_ALUNO, aluno);
-	}
+		
+		Aluno alunoModel = alunoRepository.getOne(aluno.getMatricula());
+		if (alunoModel != null) {
 
+			alunoModel.setCpf(aluno.getCpf());
+			alunoModel.setEmail(aluno.getEmail());
+			alunoModel.setFormaIngresso(aluno.getFormaIngresso());
+			alunoModel.setNome(aluno.getNome());
+			alunoModel.setTurma(aluno.getTurma());
+			
+			alunoRepository.save(alunoModel);
+		}
+		return alunoModel;
+	}
+	
+	public Aluno criarAlunoModel(AlunoDto aluno) {

```

Comment:
Deve-se renomear o método para que reflita corretamente o que faz
```suggestion
	public Aluno criarAluno(AlunoDto aluno) {
```

Reply:
feito